samples/04_gis_analysts_data_scientists/model_explainability_using_shap_for_tabular_data.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<p>we might want to change the login to <code>gis=GIS('home')</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Done</p>